src/NLog/Layouts/LayoutRenderOptions.cs

```diff
@@ -49,5 +49,9 @@ public enum LayoutRenderOptions
         /// Layout renderer method can handle concurrent threads
         /// </summary>
         ThreadSafe = 1,
+        /// <summary>
+        /// Layout renderer method is agnostic to current thread context
```

Comment:
I would be nice if we could describe this without the word "agnostic"

Reply:
Tried to improve the xml-docs